docs/installation.rst

```diff
@@ -30,6 +43,13 @@ Installation via pip
 
     pip install ansible-rulebook ansible ansible-runner
 
+.. note::
+
+    ansible-rulebook installs by default the package psycopg2-binary which is a PostgreSQL database adapter precompiled for the most common platforms.
+    For production environments, it is recommended to install the psycopg2 package instead, which requires the PostgreSQL development headers to be installed on the system.
+    Look at the `psycopg2 installation instructions <https://www.psycopg.org/psycopg3/docs/basic/install.html#local-installation>`_ for more information.
```

Comment:
I'm curious why it's called the psycopg2 package, but the link goes to psycopg3 docs?

Reply:
It's my bad. I updated it. Thanks for the warn